### PR TITLE
Adjustments to offer announcement banner for hosted Mender

### PR DESCRIPTION
**Pull request by michaelatmender**
Updated with Remote terminal link, and it should now be possible to hide/show for different plans + trial tier if needed.

### CODE REVIEW

File: src/js/components/header/header.js [hunk 1]
```diff
@@ -83,11 +94,23 @@ export class Header extends React.Component {
...
+  }
+
   onLogoutClick() {
     this.setState({ gettingUser: false, loggingOut: true, anchorEl: null });
     this.props.logoutUser();
   }
 
+  setHideOffer() {
+    const self = this;
+    this.cookies.set('offer', 'remote', { path: '/', maxAge: 2629746 });
```
Comment (mzedel) [inline src/js/components/header/header.js:110]:
```suggestion
    this.cookies.set('offer', currentOffer.name, { path: '/', maxAge: 2629746 });
```

**Review by mzedel — APPROVED** [review]
looks good! one small thing I would change, that might get overlooked otherwise...

File: src/js/components/header/header.js (hunk 1 above)
Comment (michaelatmender) [reply src/js/components/header/header.js]: Thanks for catching that!

File: src/js/components/header/offerheader.js [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+    {organization && organization.trial ? (
+      <span>
+        Try out the new <b>Remote terminal</b> feature for free with your current trial.&nbsp;
+      </span>
+    ) : (
+      <span>
+        Try out the new <b>Remote terminal</b> feature for free on your current plan until March 31st.&nbsp;
+      </span>
+    )}
+    <a href="https://docs.mender.io/add-ons/remote-terminal" target="_blank" rel="noreferrer">
```
Comment (mzedel) [inline src/js/components/header/offerheader.js:16]: based on the tests I think this lacks:
```suggestion
    <a href={`https://docs.mender.io/${docsVersion}add-ons/remote-terminal`} target="_blank" rel="noreferrer">
```

**Review by tranchitella — APPROVED** [review]

**Pull request merged**